stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/ChatViewModel.kt

```diff
@@ -38,6 +39,8 @@ class ChatViewModel(private val cid: String, private val chatDomain: ChatDomain
         }
     }
 
+    internal fun getChannelName(): String? = channelController?.toChannel()?.name
```

Comment:
Date flows the wrong way around here - we're pulling it into the Fragment from the ViewModel, instead of the ViewModel exposing it to be observed.

`MessageListViewModel` already exposes the `Channel` that it's displaying messages for, maybe we can observe that?

Reply:
Let me add a mediator here then